src/Tribe/Views/V2/Hooks.php

```diff
@@ -635,4 +636,15 @@ public function filter_get_stylesheet_option( $value, $key ) {
 
 		return $value;
 	}
+
+	/**
+	 * Print Single Event JSON-LD.
+	 *
+	 * @since TBD
+	 *
+	 */
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 97c6107252983a04f98ab4c16719e589</sub>